lms/djangoapps/grades/migrations/0001_initial.py

```diff
@@ -37,6 +60,7 @@ class Migration(migrations.Migration):
                 ('id', models.AutoField(verbose_name='ID', serialize=False, auto_created=True, primary_key=True)),
                 ('blocks_json', models.TextField()),
                 ('hashed', models.CharField(unique=True, max_length=100)),
+                ('course_id', xmodule_django.models.CourseKeyField(max_length=255, db_index=True)),
```

Comment:
For posterity - this line is the only change that isn't present in the now-squashed 0002 and 0003 migrations